frontend/src/locales/fr.json

```diff
@@ -584,7 +584,9 @@
   "pvp": {
     "lastDefeated": "Vous avez été vaincu pour la dernière fois par :",
     "tapAnywhere": "Appuyez n'importe où pour fermer",
-    "kicked": "Vous avez été expulsé de l'arène !"
+    "kicked": "Vous avez été expulsé de l'arène !",
```

Comment:
Just so you know, you don't need to provide all the translations :D There's always fallback for english, and translations can be handled by translators. Nice addition nevertheless!

Reply:
Too late! Got most of the translations ready in another PR, should I delete them and leave them to the translators or leave them in? I would assume some of them might be wrong due to Google translate, but I don't know what's easier for them :o

Reply:
Just leave it however you've done it :D Not a problem for them, just to assure you, you don't need to provide ALL translations, just english, that's why I was surprised you guys didn't want to do them on the spot. It's just adding a key value in en.json, the rest will be handled.